### PR TITLE
Feat 1 init

### PR DESCRIPTION
**Pull request by glennfriend**
#1

### CODE REVIEW

**Comment by glennfriend** [comment]
[add loggly client](https://trello.com/c/OpRxdrru/277-add-loggly-client)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- vi .bablerc
+```
+{
+  "plugins": [
+    [
+      "module-resolver",
+      {
+        "root": ["./"],
+        "alias": {
+          "universal-loggly": "./src/lib/universal-loggly-helpers",
```
Comment (koshuang) [inline README.md]: 如果是 npm project，應該是不用設定 module-resolver

**Comment by koshuang** [comment]
r+, thanks!

File: src/LogglyClient.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ *  可能無法發送訊息的原因
+ *    - token 有問題
+ *    - 被 browser 的廣告 plugin 阻止
+ */
+export class LogglyClient {
+  logglyLoggerFormat;
+
+  constructor(token, tags) {
+    this.logglyLoggerFormat = new LogglyLoggerFormat(token, tags);
```
Comment (koshuang) [inline src/LogglyClient.ts]: 一般命名會取成名詞，例如 `LogglyLoggerFormatter`

**Comment by koshuang** [comment]
r+ with a comment, thanks!

**Pull request merged**